react/features/whiteboard/functions.ts

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable import/order */
+import md5 from 'js-md5';
+
+// @ts-ignore
+import { getPinnedParticipant } from '../../features/base/participants';
+import { appendURLParam } from '../base/util/uri';
+
+// @ts-ignore
+import { getCurrentRoomId, isInBreakoutRoom } from '../breakout-rooms/functions';
+
+const getWhiteboardState = (state: any): any => state['features/whiteboard'];
+
+/**
+ * Indicates whether the whiteboard is open.
+ *
+ * @param {Object} state - The state from the Redux store.
+ * @returns {boolean}
+ */
+export const isWhiteboardOpen = (state: any): boolean => getWhiteboardState(state).enabled;
+
+/**
+ * Returns the whiteboard id.
+ *
+ * @param {Object} state - The state from the Redux store.
+ * @returns {string|null}
+ */
+export const getWhiteboardId = (state: any): string | null => getWhiteboardState(state).id;
```

Comment:
Why do we need this? We should be able to deterministically resolve it from the room name, shouldn't we?